### DISCUSSION TITLE
Focus split (option) for clicks on the `channel is live!` Chatterino toast notification

### DISCUSSION THREAD

**Discussion by brian6932** [Chatterino/chatterino2#4887] [Ideas]
If a split exists that matches the live channel on the toast notification, it would be nice if Chatterino would focus that split, as well as, for example, open their channel in Streamlink on click.
If the live channel doesn't have a matching split, either a new split could be created in the current tab, or just ignore it and continue with the other click actions.

**Comment by 8thony** [comment]
- Probably duplicate of #4910

You might want to add your idea to the linked discussion and give it an upvote.

**Reply by brian6932** [reply]
Not really a duplicate, that's a pretty general request, and doesn't mention split focusing specifically.
